### PR TITLE
Add hyperlink to transcripts & restore missing loc

### LINKED ISSUE #249: Display transcript directory (`loc`) on Transcripts Page
The `loc` information, once directly accessible, is no longer displayed on the transcripts page due to recent code changes. While this information is still derivable from existing data, it now requires additional computation and is not readily available to users.

### LINKED ISSUE #250: Add Hyperlink to Transcripts on Transcripts Page
With the recent refactoring, we now have access to the link of a queued transcript at `btctranscripts.com/<loc>/<filename>`. To enhance user experience and accessibility, I propose adding a hyperlink to the transcript titles on the transcripts page. Clicking on the title should redirect users (in a new tab) to the corresponding transcript link on btctranscripts.com.

### PR DESCRIPTION
**Pull request by kouloumos**
Fixes #249 
Fixes #250

### CODE REVIEW

File: src/components/tables/TitleWithTags.tsx [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
   transcriptsCategories,
 } from "@/utils";
-import { Box, Flex, Td, Text } from "@chakra-ui/react";
+import config from "@/config/config.json";
+import { resolveGHBranchUrl } from "@/utils/github";
+
+
+const resolveTranscriptUrl = (transcriptUrl: string | null) => {
+  if (transcriptUrl) {
+    const { path, dir } = resolveGHBranchUrl(transcriptUrl)
```
Comment (Emmanuel-Develops) [inline src/components/tables/TitleWithTags.tsx:15]: @kouloumos, there are some dead code in that file from when we were still going back and forth on the URL format. `resolveGHBranchUrl` is one of them.

the function to resolve all githuburl in the db i.e `transcriptUrl` or `branchUrl` is `resolveGHApiUrl`
https://github.com/bitcointranscripts/transcription-review-front-end/blob/d6aeacab216c3371de7170cc7c919942899ee605/src/utils/github.ts#L248

**Review by Emmanuel-Develops — CHANGES_REQUESTED** [review]

**Comment by kouloumos** [comment]
@Emmanuel-Develops force-push with suggested change

**Review by Emmanuel-Develops — APPROVED** [review]

File: src/components/tables/TitleWithTags.tsx [hunk 2]
```diff
@@ -31,14 +47,20 @@ const TitleWithTags = ({
...
     <Td width="40%">
       <Flex gap={2} flexDir="column">
         <Box>
-          <Text>{title}</Text>
+          {!transcript && <Text>{title}</Text>}
+          {transcript && (
+            <Link target="_blank" rel="noopener" href={transcript.url}>
+              <Text>{title}</Text>
+            </Link>
+          )}
```
Comment (BalogunofAfrica) [inline src/components/tables/TitleWithTags.tsx:61]: This can be expressed as a ternary

Comment (BalogunofAfrica) [reply src/components/tables/TitleWithTags.tsx:61]: Alright LGTM then

**Review by BalogunofAfrica — APPROVED** [review]

**Pull request merged**